src/components/RemoveToDo.vue

```diff
@@ -1,15 +1,24 @@
 <template>
-  <button @click="removeTodo">x</button>
+  <button class="delete-button" @click="removeTodo">Delete</button>
 </template>
 
 <script setup lang="ts">
-
 const emit = defineEmits<{
-  (e: 'todoRemoved', type: any): void
-}>()
+  (e: "todoRemoved", type: any): void;
+}>();
 
 function removeTodo() {
   emit("todoRemoved", "");
 }
-
-</script>
\ No newline at end of file
+</script>
+<style scoped>
+.delete-button {
+  width: 119.64px;
+  height: 52.05px;
+  left: 570.36px;
+  top: 472.94px;
```

Comment:
Please try to not copy paste all the style from the figma avoid as much as you can to add a fix width and height to the elements it breaks the responsiveness of the page
Please review this across all your files I see this approach multiple times